guardian/src/main/java/org/rfcx/guardian/guardian/manager/CredentialKeeper.kt

```diff
@@ -10,6 +10,7 @@ class CredentialKeeper(val context: Context) {
         // Required
         preferences.putString(PreferenceManager.USER_GUID, user.guid)
         preferences.putString(PreferenceManager.ID_TOKEN, user.idToken)
+        preferences.putLong(PreferenceManager.EXPIRED_AT, user.expiredAt)
```

Comment:
Should it be TOKEN_EXPIRED_AT? So when we look at all the preferences we know this relates to the token.

Reply:
Agree with that